server/utils/referrals/caseListUtils.ts

```diff
@@ -103,7 +103,11 @@ export default class CaseListUtils {
     })
   }
 
-  static tableRowContent(referralSummary: ReferralSummary, column: CaseListColumnHeader): string {
+  static tableRowContent(
+    referralSummary: ReferralSummary,
+    column: CaseListColumnHeader,
+    paths: typeof assessPaths | typeof referPaths = assessPaths,
```

Comment:
I'd be tempted to remove the default parameter here, so there's no risk of people forgetting it and/or interpreting a priority of one over the other

Reply:
We'd need to provide it even when it's irrelevant (i.e. for any other column) - do you think that's better than accidentally using a default?

Reply:
Hmm, I forgot how widespread our use of `tableRowContent` was! I wonder if we should have a look at breaking it down slightly as a refactoring task, rather than blocking this PR.